lib/utils/MyLocalizations.dart

```diff
@@ -20,9 +20,29 @@ class MyLocalizations {
 
   static Future<MyLocalizations> loadTranslations(Locale locale) async {
     var appTranslations = MyLocalizations(locale);
-    var jsonContent = await rootBundle.loadString(
-      'assets/language/${locale.languageCode}_${locale.countryCode}.json'
-    );
+    String? jsonContent;
+    final regionFilename = 'assets/language/${locale.languageCode}_${locale.countryCode}.json';
+
+    try {
+      // Try loading the region-specific file
+      jsonContent = await rootBundle.loadString(regionFilename);
+    } catch (_) {
+      // If not found, load the language from any region
+      final supportedLanguages = ['bg_BG', 'bn_BD', 'ca_ES', 'de_DE', 'el_GR', 'en_US', 'es_ES', 'es_UY', 'eu_ES',
+        'fr_FR', 'gl_ES', 'hr_HR', 'hu_HU', 'it_IT', 'lb_LU', 'mk_MK', 'nl_NL', 'pt_PT', 'ro_RO', 'sl_SI',
+        'sq_AL', 'sr_RS', 'sv_SE', 'tr_TR',
```

Comment:
I think it's not ideal or good to have the codes here because they are also hardcoded in settings_page.dart. However, loading the files using the directory was not working as `assets/language` was not being detected (I don't know why, since it's added to pubspec.yaml and the directory should be correct)